sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/page_blob_client.hpp

```diff
@@ -182,7 +182,7 @@ namespace Azure { namespace Storage { namespace Blobs {
      * @param range Specifies the range of bytes to be cleared. Both the start and end of the range
      * must be specified. For a page clear operation, the page range can be up to the value of the
      * blob's full size. Given that pages must be aligned with 512-byte boundaries, the start of the
-     * range must be a modulus of 512 and the end of the range must be a modulus of 512 � 1.
+     * range must be a modulus of 512 and the end of the range must be a modulus of 512 � 1.
```

Comment:
what is this?

Reply:
Probably some encoding issue with a custom modulo operator. We should fix it to use plaintext `%` (or other symbol if I a misinterpreting what was meant here).

You added this here: https://github.com/Azure/azure-sdk-for-cpp/pull/1247/